src/current/cockroachcloud/custom-metric-chart-page.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Custom Metric Chart Page
+summary: The Custom Metric Chart page lets you create one or multiple custom charts showing the time series data for an available metric or combination of metrics.
+toc: true
+---
+
+The **Custom Metric Chart** page is available for CockroachDB {{ site.data.products.serverless }} clusters. To view this page, select a cluster from the [**Clusters** page]({% link cockroachcloud/cluster-management.md %}#view-clusters-page), and click **Metrics** in the **Monitoring** section of the left side navigation. Navigate to the **Custom** tab to create one or multiple custom charts showing the time series data for an available metric or combination of metrics.
+
+{{site.data.alerts.callout_info}}
+{% include_cached feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+## Use the Custom Metric Chart page
+
+<img src="{{ 'images/cockroachcloud/custom-metric-chart.png' | relative_url }}" alt="Custom Metric Chart" style="border:1px solid #eee;max-width:100%" />
+
+On the **Custom Metric Chart** page, you can set the time span for all charts, add new custom charts, and customize each chart:
```

Comment:
Consistency Nit: Is time 'span' in our docs? Always thought it was time range (e.g., "time range").

Reply:
updated. You are correct: docs use time range or time interval.